### PR TITLE
feat: Support Bluesky

### PR DESCRIPTION
**Pull request by kmackenzieii**
*Ensure your PR title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec*

## Motivation

Bluesky is becoming more popular. Now is a good time to add a bot

See feature request ticket #46 

## Changes

added support for automated posting to Bluesky in the same style as other bots. 

added dependency on [AT Protocol SDK for Python](https://atproto.blue/en/latest/index.html)

See post from [test bot](https://bsky.app/profile/25d6f49a955b8b1d.bsky.social)

## Testing Instructions

Create an account on [Bluesky](https://bsky.app/).

Create an app password _optional_ 

Export the following environment variables
`ATP_PDS_HOST`=the url of the pds server if self hosting, will default to bluesky's server _optional_ 
`ATP_AUTH_HANDLE`=the handle for the account
`ATP_AUTH_PASSWORD`=the password or app password for the account

run as usual, i.e. `poetry run python3 slowzones.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ![lint](https://github.com/transitmatters/mbta-slow-zone-bot/workflows/lint/badge.svg?branch=main)
 ![run](https://github.com/transitmatters/mbta-slow-zone-bot/workflows/run/badge.svg?branch=main)
 
-A Twitter & Mastodon & Slack bot to track MBTA slow zones
+A Twitter & Mastodon & Slack & Bluesky bot to track MBTA slow zones
```
Comment (nathan-weinberg) [inline README.md:5]: It does it all!

File: slowzones.py [hunk 2]
```diff
@@ -42,6 +47,9 @@
     access_token=MASTODON_ACCESS_TOKEN,
 )
 
+bluesky_client = atproto.Client(base_url=ATP_PDS_HOST)
+bluesky_client.login(login=ATP_AUTH_HANDLE, password=ATP_AUTH_PASSWORD)
```
Comment (nathan-weinberg) [inline slowzones.py:51]: This seems to be what's failing the dry run: https://github.com/transitmatters/mbta-slow-zone-bot/actions/runs/11939535356/job/33280264410?pr=53

Maybe this client login should be moved under L126 so it's captured by our existing exception?

If so we should follow suit with the other domains, but that can be a separate PR.

**Review by nathan-weinberg — COMMENTED** [review]
Overall looking good to me! Couple comments

We'll get the creds added to the GitHub account before merging :+1:

File: slowzones.py [hunk 3]
```diff
@@ -24,6 +26,9 @@
 MASTODON_CLIENT_KEY = os.environ.get("MASTODON_CLIENT_KEY")
 MASTODON_CLIENT_SECRET = os.environ.get("MASTODON_CLIENT_SECRET")
 MASTODON_ACCESS_TOKEN = os.environ.get("MASTODON_ACCESS_TOKEN")
+ATP_PDS_HOST = os.environ.get("ATP_PDS_HOST") or None
```
Comment (devinmatte) [inline slowzones.py:29]: This is not being defined for the CI, meaning dry run or otherwise can't run

**Review by devinmatte — CHANGES_REQUESTED** [review]

**Review by devinmatte — APPROVED** [review]

**Pull request merged**